tests/integration/test_sqs.py

```diff
@@ -906,6 +906,40 @@ def test_list_queue_with_auth_in_presigned_url(self):
         self.assertEqual(res.status_code, 200)
         self.assertTrue(b'<ListQueuesResponse>' in res.content)
 
+    def test_get_list_queues_with_auth_in_presigned_url(self):
```

Comment:
Thanks much for this follow-up PR @seveibar ! 👍  

nitpick: Looks like there is quite some duplication in the two test methods - do you think we could move the common parts into a util method `list_queues_with_auth_in_presigned_url(self, method)` and then call this method from the two test methods with `method='POST'` and `method='GET'`? This would make things a bit cleaner. Thanks!

Reply:
will do!!